CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main (unreleased)
 
+## 2.0.0 (2022-03-11)
```

Comment:
Probably won't get a review today, so I put in tomorrow's date for release. This should be updated before merging, if the release is on a different day.